### PR TITLE
NETOBSERV-1764 Frontend code styling follow-up

### PR DESCRIPTION
**Pull request by jpinsonneau**
## Description

Followup on frontend refactor to move fetch function in their respective components.

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [X] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [X] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: web/src/components/netflow-traffic.tsx [hunk 1]
```diff
@@ -101,6 +81,7 @@ export const NetflowTraffic: React.FC<NetflowTrafficProps> = ({ forcedFilters, i
 
   // Refs
   const metricsRef = React.useRef(model.metrics);
+  const drawerRef = React.useRef<NetflowTrafficDrawerHandle>(null);
```
Comment (jpinsonneau) [inline web/src/components/netflow-traffic.tsx:84]: a single ref for the drawer holds overview, table and topology refs

File: web/src/components/drawer/netflow-traffic-drawer.tsx [hunk 2]
```diff
@@ -93,314 +100,332 @@ export interface NetflowTrafficDrawerProps {
...
-      props.setSelectedElement(element);
-    },
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    [props.clearSelections, props.setSelectedElement]
-  );
+    React.useImperativeHandle(ref, () => ({
+      getOverviewHandle: () => overviewRef.current,
+      getTableHandle: () => tableRef.current,
+      getTopologyHandle: () => topologyRef.current
+    }));
```
Comment (jpinsonneau) [inline web/src/components/drawer/netflow-traffic-drawer.tsx:114]: Each of these are binded to their respective component

**Comment by jpinsonneau** [comment]
Fixed panel view deps
https://github.com/netobserv/network-observability-console-plugin/compare/681daaa84cd2df6f1ec1f31ff51019d749feac51..abbaf9a4c306e85145a1681e4589cfc63def4d14

**Comment by Amoghrd** [comment]
@jpinsonneau Ran all automated frontend regression tests on PR [cypress-tests#724](https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/cypress-tests/724/) Some failures are flakes and passed on rerun except the netflow_table test which is failing on the histogram step. On rerun too see the same issue where the histogram does not load the first time table tab is visited, if page is refreshed or if we visit another tab and come back then it is visible. Possibly some race condition issue. Could you PTAL? TIA!

**Comment by jpinsonneau** [comment]
> @jpinsonneau Ran all automated frontend regression tests on PR [cypress-tests#724](https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/cypress-tests/724/) Some failures are flakes and passed on rerun except the netflow_table test which is failing on the histogram step. On rerun too see the same issue where the histogram does not load the first time table tab is visited, if page is refreshed or if we visit another tab and come back then it is visible. Possibly some race condition issue. Could you PTAL? TIA!

Good catch @Amoghrd  ! I was missing some dependencies on the function running the queries. I have added the histogram ones but also some overview / topology related options
https://github.com/netobserv/network-observability-console-plugin/pull/558/commits/c5a85ebb153acf64d472b712828d3342def3057f

It's querying the histogram at first run now on my cluster.

Please let me know if you find other issues while playing with the UI.

**Comment by Amoghrd** [comment]
Reran frontend automated regression tests. Barring some flakes all are passing now [cypress-tests#739](https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/cypress-tests/739/console)
/label qe-approved

**Review by jotak — APPROVED** [review]

**Pull request merged**